test/inspector/test-async-hook-teardown-at-debug-end.js

```diff
@@ -7,23 +7,12 @@ const spawn = require('child_process').spawn;
 
 const script = `
 const assert = require('assert');
+const async_wrap = process.binding('async_wrap');
+const {kTotals} = async_wrap.constants;
```

Comment:
I think if this were a “real” script out linter would expect spaces inside `{}` … no big deal, tho :)